db.php

```diff
@@ -11,52 +11,53 @@ function getItems(object $db): array {
 function arraySplit(array $items) : string{
```

Comment:
Charlie might ask you to rename this file from db.php to functions.php

Reply:
Also I would add docblocks for your functions

Reply:
I have added doc blocks to my functions